packages/amplify-category-api/src/graphql-transformer/transform-graphql-schema-v2.ts

```diff
@@ -302,12 +302,19 @@ const getDatasourceSecretMap = async (context: $TSContext): Promise<Map<string,
   return outputMap;
 };
 
-const getSQLLambdaVpcConfig = async (context: $TSContext): Promise<VpcConfig | undefined> => {
+const getSQLLambdaVpcConfig = async (context: $TSContext): Promise<VpcSubnetConfig> => {
```

Comment:
Although you intend to use this for Lambda, you're really getting the VPC config for the RDS host here, right? A rename might clarify that your intent is to install the Lambda into the RDS host's VPC. I can see an eventual requirement to install the Lambda into its own VPC, and let customers set up a private link between the Lambda VPC & the RDS VPC, so the rename will make it clearer to future devs (i.e., us) where that change needs to happen :)